packages/next-tinacms-cloudinary/src/handlers.ts

```diff
@@ -122,34 +123,37 @@ async function listMedia(
     }
     let folders: string[] = []
     let folderRes = null
+    const loadFolders = !(filesOnly == 'true')
 
-    try {
-      // @ts-ignore
-      folderRes = await cloudinary.api.folders(directory)
-    } catch (e) {
-      // If the folder doesn't exist, just return an empty array
-      if (e.error?.message.startsWith("Can't find folder with path")) {
-        // ignore
-      } else {
-        console.error('Error getting folders')
-        console.error(e)
-        throw e
+    if (loadFolders) {
```

Comment:
Can this be inverted? i.e. if(!loadFolders) return;

Reply:
Good idea